app/modules/projects/projects_controller.py

```
@@ -1,24 +1,129 @@
+            
+            # Create project
+            project = Project(
+                id=project_id,
+                user_id=user_id,
+                properties=properties,
+                status=ProjectStatusEnum.SUBMITTED.value,
+                repo_name="",  # Will be set when repository is selected
+                branch_name="",  # Will be set when repository is selected
+            )
+            
+            project_service = ProjectService(db)
```

Ruff (0.14.10)

Local variable `project_service` is assigned to but never used

Remove assignment to unused variable `project_service`

(F841)